### PR TITLE
CentOS Stream 8 and Rocky Linux 8

### PR DESCRIPTION
**Pull request by matyasselmeci**
- Name the CentOS Stream 8 image centos_stream_8 instead of centos_8_stream
- Add Rocky Linux 8.4

### CODE REVIEW

File: rocky_8/kickstart.ks [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#auth --enableshadow --passalgo=sha512 --kickstart
+# For some reason libguest can't detect the default (xfs)
+# file system for CentOS 7 installs;  Overridden to use
+# ext4 - Moate
+autopart --type=lvm --fstype=ext4
```
Comment (brianhlin) [inline rocky_8/kickstart.ks:5]: Do you know if we still need this for EL8?

**Review by brianhlin — APPROVED** [review]
One question, pre-approving

File: rocky_8/kickstart.ks (hunk 1 above)
Comment (matyasselmeci) [reply rocky_8/kickstart.ks:5]: No idea, never tried. The CentOS docs say that 8 also uses XFS as the default file system so unless libguest has been fixed, we'll need it there too.

Comment (matyasselmeci) [reply rocky_8/kickstart.ks]: We don't appear to need it for 8.

**Pull request merged**